### DISCUSSION TITLE
Soup failed by update from 2.3.270 to 2.3.280

### DISCUSSION THREAD

**Discussion by Nippler1** [Security-Onion-Solutions/securityonion#12028] [Unsupported Versions]
Hi, 
i make an soup update from Version 2.3.270 to 2.3.280  on an standalone SO System installed with an Security Onion ISO image and i have the following soup error (soup.log):

#########
Summary for local
 --------------
Succeeded: 745 (changed=49)
Failed:      2
 --------------
Total states run:     747
Total run time:   648.820 s
 
Starting crond service at 06:47:53.147980
Successfully started crond.
Starting salt-master service at 06:47:53.209815
Successfully started salt-master.
Starting salt-minion service at 06:47:53.244407
Successfully started salt-minion.
Enabling highstate.
local:
      ----------
      msg:
      Info: highstate state already enabled.
      res:
      True

Soup failed with error 1: Unhandled error
Unhandled error occured, please check /root/soup.log for details.
###########

It looks like there was a problem with elk, 4 times in the soup.log:

###########
             ID: wait_for_elasticsearch
  Function: cmd.run
      Name: so-elasticsearch-wait
      Result: False
Comment: Command "so-elasticsearch-wait" run
    Started: 06:40:16.233318
 Duration: 369260.986 ms
 Changes:   
              ----------
              pid:  
                  38391 
              retcode:
                  1     
              stderr:
              stdout:
                  Waiting for value 'green open' at 'https://localhost:9200/_cat/indices/.kibana*' (1/300)
                  Server is not ready 
                  Waiting for value 'green open' at 'https://localhost:9200/_cat/indices/.kibana*' (2/300)
                  Server is not ready 
                  Waiting for value 'green open' at 'https://localhost:9200/_cat/indices/.kibana*' (3/300)
                  .
                  .
                  Server is not ready
                  Server still not ready after 300 attempts; giving up.
############

After i reboot the system,  salt-call state.highstate run without errors und the SOC looks fine.
But the ssl Certificat issue with Chrome is not fixed. We have the same Chrome error (ERR_SSL_KEY_USAGE_INCOMPATIBLE) like before on Version 2.3.270.

How can i fix it? It is possible to run the complete update again?

Many Thanks for your Help.

**Comment by dougburks** [comment]
You can try running soup again. However, please keep in mind that 2.3 reaches End Of Life in less than 4 months so your time may be better spent migrating to 2.4:
https://blog.securityonion.net/2023/12/4-month-end-of-life-eol-reminder-for.html
